hw2/aes_ttable_cache_attack.py

```diff
@@ -169,9 +183,18 @@ def find_unviable_candidates(j, T_result, k0_low, viable):
     k0_2 = k0_low >> 4 & 0xf
     k0_3 = k0_low & 0xf
 
+    k0_list = [k0_0, k0_1, k0_2, k0_3] # !!!!!!! perhaps opposite order

```

Comment:
We can actually remove the `k0_list`, since we anyways access each index explicitly in line 193.